src/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List</title>
+</head>
+<body>
+  
+  <ul class="list-container p-5">
+     <li class="list-group-item"><h6 class="h6">Today's To Do</h6></li>
+    <form id="add-todo">
+      <input class="form-control list-group fst-italic" id="add-todo-input" type="text"  placeholder="   Add to your list..." required>
+    
+    </form>
+    
+  </ul>
+  
+</body>
+</html>
```

Comment:
💡  It's best practice to end each file with an empty line, according to best practices.